bin/mastarm

```diff
@@ -9,7 +9,7 @@ const util = require('../lib/util')
 commander
   .version(require('../package.json').version)
   .option('-c, --config <path>', 'Path to configuration files.', path.join(process.cwd(), '/configurations/default'))
-  .option('-e, --env <environment>', 'Environment to use.', process.env.NODE_ENV || 'development')
+  .option('-e, --env <environment>', 'Environment to use.')
```

Comment:
I'm not sure what you mean by this. If it breaks the SE tests than we could just set the flag manually.

I removed the default because it throws off how we `get` the environment later on. It's too early of an override.